cranelift/isle/fuzz/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "isle-fuzz"
```

Comment:
FWIW I don't think this will run on oss-fuzz unless it's in the top-level directory by default (but we could modify oss-fuzz to build here too if desired)